libs/api-interfaces/src/lib/aggregate-views/view-models/resources/media-items/mime-type.enum.ts

```diff
@@ -3,6 +3,7 @@ export enum MIMEType {
     // documents
     pdf = 'application/pdf',
     csv = 'text/csv',
+    txt = 'text/plain',
```

Comment:
Do this on a separate branch.

Reply:
This has been marked with a TODO.

Reply:
[This](https://coscrad.atlassian.net/browse/CWEBJIRA-296) is the relevant story.